extensions/os_cmd.js

```diff
@@ -8,30 +8,32 @@ const {exec} = require("child_process");
 const {execSync} = require("child_process");
 const {Ticketing} = require(`${CONSTANTS.LIBDIR}/Ticketing.js`);
 
+const _toBuffer = (pid, data) => Buffer.concat([Buffer.from(`[PID:${pid}]`), Buffer.from(data, 'binary')]);
```

Comment:
Small nitpick but we use double quotes as a convention so this should be "binary" not 'binary'.